content/en/docs/how-tos/cluster-claim.md

```diff
@@ -212,11 +253,27 @@ document.
 
 ## Troubleshooting Cluster Pools
 
+### Access Cluster Installation logs
 The cluster pools are maintained by Hive behind the scenes, so installation failures, cloud platform account
 misconfigurations and similar issues are not exposed to actual CI jobs: the jobs will simply never successfully claim a
-cluster if Hive fails to install them. Currently, cluster pool owners cannot access the logs or other troubleshooting
-information on the `hive` cluster. For now, work with DPTP (ping `@dptp-helpdesk` on #forum-testplatform Slack channel)
-to troubleshoot pools that never get ready clusters.
+cluster if Hive fails to install them.
+
+The installation logs can be found in the `hive` container logs with the following commands:
+
+```console
+# filter out the namespaces start with the cluster pool's name
+$ oc get ns | grep dptp-demo-cluster-pool
```

Comment:
Do these not get annotations or labels or anything to do something more intelligent for filtering here?

Reply:
Perfect! It has the label.

```console
oc --context hive get ns ci-ocp-4-8-amd64-aws-us-east-1-b94kw -o yaml
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    openshift.io/sa.scc.mcs: s0:c39,c9
    openshift.io/sa.scc.supplemental-groups: 1001500000/10000
    openshift.io/sa.scc.uid-range: 1001500000/10000
  creationTimestamp: "2021-07-01T19:09:09Z"
  labels:
    hive.openshift.io/cluster-pool-name: ci-ocp-4-8-amd64-aws-us-east-1
    kubernetes.io/metadata.name: ci-ocp-4-8-amd64-aws-us-east-1-b94kw
```

Reply:
I will update tomorrow.

Reply:
Done.